extensions/search-npm/package.json

```diff
@@ -10,7 +10,7 @@
     {
       "name": "index",
       "title": "Search Packages",
-      "subtitle": "npm",
+      "subtitle": "Search for npm package information",
```

Comment:
Previous subtitle was correct. It's rather for context than for description. Please check naming guidelines [here](https://developers.raycast.com/basics/prepare-an-extension-for-store).

Reply:
@PitNikola I have made the change now